weasyl/middleware.py

```diff
@@ -175,7 +175,7 @@ def _generate_http2_server_push_headers():
         d.cdnify_url('/static/scripts.js?' + d.CURRENT_SHA),
     ]
     js_preload = ", ".join('<' + item + '>; rel=preload; as=script' for item in js_resource_preload_list)
-    return ", ".join([js_preload, css_preload])
+    return ", ".join([js_preload, css_preload]).encode('latin-1')
```

Comment:
How was this a lint error?

Reply:
Well, lint in the sense that it was raised by the ``../../../virtualenv/python2.7.14/lib/python2.7/site-packages/webtest/lint.py`` package; I suspect that Nginx/Cloudflare would still process it without issue, but this was more, "let's satisfy Webtest". (I admit, the commit description could have been a bit more descriptive; I'll ``--amend`` this one if you'd like.)

https://travis-ci.org/Weasyl/weasyl/jobs/362430306#L1285